content/docs/contributing/docs.mdx

```diff
@@ -0,0 +1,274 @@
+|-- community/              # Community related content (events, general information, etc.)
+|-- docs/                   # Unikraft documentation, split by categories into subdirectories
+|-- guides/                 # Short and on point guides, good for getting started
+|-- hackathons/             # Hackathon landing pages
+|-- imprint.md
+|-- privacy.md
+`-- releases/               # Unikraft releases summaries
+```
+
+### Testing Changes
+
+Before you submit your changes for review, please make sure you have tested them thoroughly.
+You can build and deploy the site locally, following [the instructions below](/docs/contributing/docs/#building-the-website).
+You can also validate your changes for common issues using the [`super-linter`](https://github.com/github/super-linter), following the [instructions below](docs/contributing/docs/#using-the-linter).
+Please also do a spell check on your changes.
+Most text editors should be able to do that by themselves, [here's how you do it in `Vim`](https://thoughtbot.com/blog/vim-spell-checking).
+
+#### Building the Website
+
+You can build the website inside a Docker environment or natively.
+Building inside a Docker environment is **strongly recommended**, since you will work in the same environment as the deployed website.
+
+#### Building the Website Inside a Docker Environment
+
+You can build the site inside a Docker environment.
+For this, you will need to install Docker-CE.
+You can do that by following the instructions [here](https://github.com/docker/docker-install), or by running:
+
+```console
+curl -fsSL https://get.docker.com/ | sh
+```
+
+<Info>
+Building the site inside a Docker environment is highly recommended, since you will work in the same environment as the deployed site, so there will be no errors regarding packages version or filesystem layout.
+</Info>
+
+To deploy the site, run (in the directory of the repository clone):
+
+```console
+docker build -t ghcr.io/unikraft/docs:base --target base .
+
+docker run -it --rm -v $(pwd):/docs -w /docs -p 3000:3000 --entrypoint sh ghcr.io/unikraft/docs:base
+
+yarn run dev
+```
+
+You may need admin privileges to run the first 2 commands.
+The website will be rendered at `localhost:3000`.
+
+#### Building the Website Natively
+
+You can also build the site natively.
+For a native build, you need to install `node`.
+You can do it by following the instructions [here](https://nodejs.org/en/download).
+
+<Info>
+This may lead to errors due to different `node` versions or other environment problems.
+These problems can be avoided by building the website inside a Docker environment.
+</Info>
+
+Then, to build and deploy the site, run:
+
+```console
+yarn install
+yarn run dev
+```
+
+Access the `http://localhost:3000/` URL in the browser to view the site.
+
+#### Using the Linter
+
+To automate a big part of the reviewing process, we use the [`super-linter`](https://github.com/github/super-linter) on the `docs` repository.
```

Comment:
```suggestion
To automate a big part of the reviewing process, we use the [`super-linter`](https://github.com/github/super-linter) on this repository.
```

Reply:
The content will be viewed on the website, `this` would seem like we are talking about the website, not a repository. I would leave it as it is now, if you want me to change it let me know.